hooks/androidCopyPreferencesPermissions.js

```diff
@@ -39,6 +39,13 @@ let permissions = {
     },
     CaloriesBurned: {
```

Comment:
Isn't this the same as `ActiveCaloriesBurned`? Do we need to keep both or can we remove `CaloriesBurned` and just keep `TotalCaloriesBurned` and `ActiveCaloriesBurned`?

Reply:
`CaloriesBurned` (*Both platforms*) = active calories burned only
`TotalCaloriesBurned` (*Android only*) = total calories burned